web/views.py

```diff
@@ -61,3 +61,9 @@ def health(request):
 def tpe(request):
     context = {}
     return render(request, 'tpe.html', context)
+    
+def team(request):
+	team = Team.objects.all()
+	context = {'team': team}
+	print(context)
```

Comment:
Please remove this

Reply:
This is used to get data from database team table. Removed the print() statement tho.